addr_manager.go

```diff
@@ -120,36 +180,35 @@ func (mgr *AddrManager) AddAddrs(p peer.ID, addrs []ma.Multiaddr, ttl time.Durat
 			continue
 		}
 
-		addrstr := string(addr.Bytes())
+		addrstr := addr.String()
```

Comment:
This is actually different, The code youre removing makes a static representation of the raw bytes of the address, the code youre adding formats the address into human readable format. The perf difference here is immense